src/services/guides/index.ts

```diff
@@ -17,31 +17,30 @@ export interface GuideContent {
   _id?: string;
   title: string;
   content: string;
-  categories: CategoryContent[],
-  digitalContents: CardDigitalContentResponse[],
+  categories: CategoryContent[];
+  digitalContents: CardDigitalContentResponse[];
 }
 
 export const getGuides = async () => {
   try {
-    return api.get<{ data: CardGuidesResponse[] }>(`/guides/list`);
+    return api.get<{ data: CardGuidesResponse[] }>(`/guides/`);
   } catch {
     throw new Error('Serviço não disponível');
   }
 };
 
 export const postGuides = async (cardBody: CardBodyInterface) => {
   try {
-    return api.post('/guides/register', cardBody);
+    return api.post('/guides/', cardBody);
   } catch {
     throw new Error('Serviço não disponível');
   }
 };
 
 export const getGuideWithCategoriesAndContent = async (id: string) => {
   try {
-    return api.get<{ data: GuideContent }>(`guides/getGuideWithCategoriesAndContent/${id}`);
+    return api.get<{ data: GuideContent }>(`guides/categoriesAndContent/${id}`);
```

Comment:
Não sei se este é o padrão rest.
Talvez seja 'guides/category/digitalContent/${idGuide}

Reply:
Nós perguntamos pro Júlio e ele disse que só não podia ter o `get` no início